mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/jobcluster/job/JobActor.java

```diff
@@ -1880,8 +1889,9 @@ public void refreshAndSendWorkerAssignments() {
         @Override
         public void checkHeartBeats(Instant currentTime) {
             LOGGER.trace("In WorkerManager::checkHeartBeats");
-            //// heartbeat misses are calculated as 3 * heartbeatInterval, pick 1.5 multiplier for this check interval
-            long missedHeartBeatToleranceSecs = (long) (1.5 * ConfigurationProvider.getConfig().getWorkerTimeoutSecs());
```

Comment:
Nit: I would get rid of MasterConfiguration::getWorkerTimeoutSecs and instead of rename it to MasterConfiguration::getDefaultWorkerTimeoutSeconds and have the default value only used within MantisJobWorkerMetadataImpl.